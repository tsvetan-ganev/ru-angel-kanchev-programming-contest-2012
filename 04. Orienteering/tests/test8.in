15 3
1403Mogilska w 08:42:47 09:44:59
1303Pavlov m 10:08:12 11:14:40
0901Miteva w 09:20:58 10:24:25
1002Bozhkov m 10:11:35 11:20:22
1102Stefanov m 09:34:44 10:17:06
1203Iliyanova w 10:11:47 11:04:17
0301Mihaylov m 08:46:20 09:28:42
0402Kirilov m 09:17:46 10:00:08
0503Dimitrov m 08:01:15 08:43:37
0802Halil m 09:37:33 10:44:24
0202Nikolova w 08:01:08 08:50:38
0602Iliev m 08:43:43 09:26:05
0703Stefanova w 09:08:47 10:14:05
1501Vamporov m 09:59:38 10:42:00
0101Atanasova w 08:14:51 09:23:27