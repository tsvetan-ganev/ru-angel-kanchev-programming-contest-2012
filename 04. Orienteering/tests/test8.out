03 04 05 06 11 15 
02 12 14 
02 